datasets/ecmwf-forecast/requirements.txt

```diff
@@ -1 +1,3 @@
-git+https://github.com/stactools-packages/ecmwf-forecast@1c9726e6a4f38e1c15648df6d45a5c8432ce92a9
\ No newline at end of file
+git+https://github.com/stactools-packages/ecmwf-forecast
```

Comment:
Let's pin this to a specific commit once https://github.com/stactools-packages/ecmwf-forecast/pull/5 is merged.

Reply:
Done.